src/libraries/System.Runtime/ref/System.Runtime.cs

```diff
@@ -9192,6 +9192,12 @@ public sealed partial class UnconditionalSuppressMessageAttribute : System.Attri
         public string? Scope { get { throw null; } set { } }
         public string? Target { get { throw null; } set { } }
     }
+    [AttributeUsage(AttributeTargets.Parameter, AllowMultiple = false, Inherited = false)]
```

Comment:
The full name should be used, the default for `AllowMultiple` is `false`
```suggestion
    [System.AttributeUsageAttribute(System.AttributeTargets.Parameter, Inherited=false)]
```